### ISSUE TITLE
Can't start Alice HTTP Inspector. Please add NavigatorKey to your application 

### ISSUE THREAD

**Issue by jigarparmar21** [flutter/flutter#59972]
navigatorKey has already been added to the code, still it shows error(red line) under the navigatorKey.

@override
Widget build(BuildContext context) {
return WillPopScope(
navigatorKey: _alice.getNavigatorKey(),
child: Theme(
data: ThemeData(
brightness: Brightness.light,
primarySwatch: AppColor.appColor,
primaryColor: AppColor.white,
buttonTheme: ButtonThemeData().copyWith(
buttonColor: AppColor.appColor,
textTheme: ButtonTextTheme.primary),
fontFamily: "EncodeSans",
),
child: layout()),

onWillPop: () {
  return handleBackPress();
});
}

**Comment by iapicca** [comment]
Hi @jigarparmar21 
From what I can see, the issue is related to the 3rd party plugin [alice](https://pub.dev/packages/alice) rather than to Flutter itself. Please open the issue in the dedicated [repository](https://github.com/jhomlala/alice/issues).
Closing, as this isn't an issue with Flutter itself.  
If you disagree, please write in the comments,  
providing your `flutter doctor -v`, your `flutter run -v`,  your `pubspec.yaml`
a minimal reproducible code sample that does not use 3rd party plugins,  
and I will reopen it.
Thank you

**Comment by jigarparmar21** [comment]
Is there anyone who can solve it ? if yes then please help me i cant solve it from last two days.

**Comment by jigarparmar21** [comment]
> Hi @jigarparmar21
> From what I can see, the issue is related to the 3rd party plugin [alice](https://pub.dev/packages/alice) rather than to Flutter itself. Please open the issue in the dedicated [repository](https://github.com/jhomlala/alice/issues).
> Closing, as this isn't an issue with Flutter itself.
> If you disagree, please write in the comments,
> providing your `flutter doctor -v`, your `flutter run -v`, your `pubspec.yaml`
> a minimal reproducible code sample that does not use 3rd party plugins,
> and I will reopen it.
> Thank you

Yes ,but i have run individual Alice inspector once. it works properly , but i want to integrate alice in my existing code. so i can track the live http calls in app.
I have not get any alice package in pubspec.yaml

**Comment by iapicca** [comment]
HI @jigarparmar21 
can you please provide your 
`flutter doctor -v`, 
your `flutter run -v`,
your `pubspec.yaml`
and a minimal reproducible code sample 
that does not use 3rd party plugins,
Thank you

**Comment by jigarparmar21** [comment]
> HI @jigarparmar21
> can you please provide your
> `flutter doctor -v`,
> your `flutter run -v`,
> your `pubspec.yaml`
> and a minimal reproducible code sample
> that does not use 3rd party plugins,
> Thank you

How does i have to provide you these flutter doctor -v, flutter run -v,  pubspec.yaml ?
Copy it and send  it or take a snapshot and send you?

**Comment by iapicca** [comment]
Hi @jigarparmar21 
you may provide the info listed above 
pasting those as comment, if the are not too long
or attaching those as .txt file;
for the sample code I suggest to open a public github repo
Thank you

**Comment by TahaTesser** [comment]
Without additional information we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.
